src/pages/api/trpc/[trpc].ts

```diff
@@ -0,0 +1,19 @@
+import { createNextApiHandler } from "@trpc/server/adapters/next";
+
+import { env } from "@src/env.mjs";
+import { createTRPCContext } from "@src/server/api/trpc";
+import { appRouter } from "@src/server/api/root";
+
+// export API handler
+export default createNextApiHandler({
+  router: appRouter,
+  createContext: createTRPCContext,
+  onError:
+    env.NODE_ENV === "development"
```

Comment:
Will this still work in production?

Reply:
This just means that we get extra logs whenever running on dev server. On live server the logs won't appear as verbose